package.json

```diff
@@ -114,7 +114,6 @@
 		"codecov": "^3.6.5",
 		"codelyzer": "^5.2.1",
 		"husky": "^4.2.3",
-		"ionic": "^5.4.16",
```

Comment:
Hey man! Don't forget to add the `@ionic/cli` just like we talk on the private chat 🙆‍♂